.github/workflows/pull-request.yml

```
@@ -295,7 +295,117 @@
+      - name: Cache Node.js dependencies
+        uses: actions/cache@v4
+        with:
+          path: |
+            ~/.npm
+            node_modules
+          key: ${{ runner.os }}-docker-check-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-docker-check-
+
+      - name: Check if Talawa API starts in Docker
+        run : | 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)